server/examples/go/hello/http_hello/functions.go

```diff
@@ -19,9 +20,6 @@ func (h *helloapp) GetBindings(w http.ResponseWriter, req *http.Request, claims
 }
 
 func (h *helloapp) Install(w http.ResponseWriter, req *http.Request, claims *api.JWTClaims, c *apps.CallRequest) (int, error) {
-	if c.Type != apps.CallTypeSubmit {
```

Comment:
do we have to remove this? Has anything changed?

Reply:
Since calls don't have type anymore, there is no need to check the type here. Since install is called from the server, it does not go under a subpath (e.g. `/install/submit`) but directly to the endpoint.

Does it make sense? Or should all calls (including bindings and install) go to the subpath?

@levb thoughts?